packages/lib/server/repository/organization.ts

```diff
@@ -53,6 +53,7 @@ export class OrganizationRepository {
           orgPricePerSeat: orgData.pricePerSeat,
           isPlatform: orgData.isPlatform,
         },
+        isPlatform: orgData.isPlatform,
```

Comment:
This is very important, since what is happening currently is the `isPlatform` boolean that comes from the orgData in our frontend stores the boolean in the metadata instead of setting the `isPlatform` table boolean to the given value . As a reason even after create an org for platform customers we have to manually go into db and set the `isPlatform` hook to true. The above line of code fixes that, so we don't have to manually go into the db from now on to set that book to true.